doc/classes/Polygon2D.xml

```diff
@@ -100,22 +100,22 @@
 			Path to a [Skeleton2D] node used for skeleton-based deformations of this polygon. If empty or invalid, skeletal deformations will not be used.
 		</member>
 		<member name="texture" type="Texture2D" setter="set_texture" getter="get_texture">
-			The polygon's fill texture. Use [code]uv[/code] to set texture coordinates.
+			The polygon's fill texture. Use [member uv] to set texture coordinates.
 		</member>
 		<member name="texture_offset" type="Vector2" setter="set_texture_offset" getter="get_texture_offset" default="Vector2(0, 0)">
-			Amount to offset the polygon's [code]texture[/code]. If [code](0, 0)[/code] the texture's origin (its top-left corner) will be placed at the polygon's [code]position[/code].
+			Amount to offset the polygon's [member texture]. If set to [code]Vector2(0, 0)[/code] the texture's origin (its top-left corner) will be placed at the polygon's position.
```

Comment:
```suggestion
			Amount to offset the polygon's [member texture]. If set to [code]Vector2(0, 0)[/code], the texture's origin (its top-left corner) will be placed at the polygon's position.
```

Reply:
Oh, by the way, I forgot to mention this myself, but I have no idea what "polygon's position" is referring to. That may be something worth clarifying by someone who understands how this works.